sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -104,34 +105,6 @@ object JdbcUtils extends Logging with SQLConfHelper {
     JdbcDialects.get(url).isCascadingTruncateTable()
   }
 
-  /**
-   * Returns an Insert SQL statement for inserting a row into the target table via JDBC conn.
-   */
-  def getInsertStatement(
-      table: String,
-      rddSchema: StructType,
-      tableSchema: Option[StructType],
-      isCaseSensitive: Boolean,
```

Comment:
The origin interface keep the `isCaseSensitive` is useful, even if it was not used in the code directly.
It seems we'd better add `isCaseSensitive` to the `JdbcDialect.insertIntoTable`.
Maybe some jdbc dialect is case sensitive.

Reply:
+1

Reply:
@Hisoka-X I know that the caller of `JdbcUtils.getInsertStatement` uses the config "spark.sql.caseSensitive".
Even if the origin `isCaseSensitive` is not used at all and I think the case-sensitive could decided by jdbc dialect.